ocaml/sdk-gen/common/CommonFunctions.ml

```diff
@@ -197,6 +197,27 @@ let gen_param_groups message params =
        (if expRelease = "" then overloadGroups else [params])
     )
 
+and get_minimum_allowed_role msg =
+  let get_last_role recroles =
+    let reversed_role_list = List.rev recroles in
+    let rec get_role reversed_list =
+      match reversed_list with
+      | [] ->
+          "Not Applicable"
```

Comment:
I was diffing the generated code:
- I see `Not Applicable` being attached to all `get_all_records` calls, for which I would expect the role to be `read-only`.
- Toolstack can correct me if I'm wrong, but for the calls that previously had an internal role, shouldn't the docs show the next public available one, i.e. `read-only`?